examples/gnrc_minimal/Makefile.gcc.dep

```diff
@@ -0,0 +1,5 @@
+# llvm compatibility requires yet to be released picolibc 1.4.7
+ifneq (,$(filter gnu,$(TOOLCHAIN)))
```

Comment:
You don't need this ifneq now. This file is included only if the gcc toolchain is used :)